freud/density.pyx

```diff
@@ -576,10 +576,21 @@ cdef class RDF(_SpatialHistogram1D):
     behavior of :math:`\lim_{r \to \infty} g(r)=1` is recovered. However, for
     very small systems the long range behavior of the radial distribution will
     instead tend to :math:`\frac{N-1}{N}`. In small systems, where this
-    deviation is noticeable, the ``normalize`` flag may be used to rescale the
-    results and force the long range behavior to 1. Note that this option will
-    have little to no effect on larger systems (for example, for systems of 100
-    particles the RDF will differ by 1%).
+    deviation is noticeable, the ``normalization_mode`` argument may be used to
+    rescale the results and force the long range behavior to 1. Note that this
+    option will have little to no effect on larger systems (for example, for
+    systems of 100 particles the RDF will differ by 1%).
+
+    .. note::
+        For correct normalization behavior when using
+        ``normalization_mode='exact'``, let the set of points be either: 1) the
+        same as the set of query points or 2) completely disjoint from the set
+        of query points.
+
+    .. note::
+        For correct normalization behavior when using
+        ``normalization_mode='finite_size'``, do not allow particles to be their
+        own neighbor.
```

Comment:
```suggestion
        own neighbor (:code:`exclude_ii` must be set to :code:`False`).
```

Reply:
I feel like it's valuable to have exact options to which this refers be spelled out. Helps the docs be more informative IMO.

Reply:
Good idea. I left it that way because one could give a custom neighborlist and in that case there are no query arguments, but having the exact answer for the much more common case will help.